src/main/java/seedu/mtracker/console/AddInstrumentParser.java

```diff
@@ -33,11 +34,19 @@ public static String getInstrumentNameFromUser(String instrumentType) {
     public static boolean isValidName(String name, String instrumentType) {
         boolean isValid = true;
         try {
-            if (name.isEmpty()) {
+            if (instrumentType.equals(AddForexParser.INSTRUMENT_TYPE)) {
+                if (name.length() != FX_PAIR_NAME_LENGTH) {
+                    throw new IllegalArgumentException();
+                }
+            } else if (name.isEmpty()) {
                 throw new IllegalArgumentException();
             }
         } catch (IllegalArgumentException e) {
-            ErrorMessage.displayAddInstrumentNameError(instrumentType);
+            if (instrumentType.equals(AddForexParser.INSTRUMENT_TYPE)) {
+                ErrorMessage.displayAddForexNameError();
+            } else {
+                ErrorMessage.displayAddInstrumentNameError(instrumentType);
+            }

```

Comment:
Can do this in a separate PR, the pr was suppose to be for junit specific changes.
On a separate note, should we do this name validation? Cause like stock we don't rly care if user puts "AAPL" vs "apple" cause we cant check with a known db of valid instrument names. For forex, even if we force user to put 6 characters they can still put this rubbish like aaaaaa. So I was thinking maybe we don't do this check and we just put a note on user guide that its up to the user responsibility to put in accurate info

Reply:
I still think the length check is redundant, like I write "usd to sgd" its like somewhat understandable to user. The point is like if not possible to validate a name completely I feel its pointless to add the check

Reply:
Nah man forex names are not <smthin> to <smthin>. Its always like USDSGD, AUDUSD, EURJPY, etc. so I want to keep this length check.